splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/util/Validator.js

```diff
@@ -73,14 +75,16 @@ class Validator {
                     : getFormattedMessage(15, [label, validator.pattern]),
             };
         }
+        return false;
     }
 
     // Validate the custom component
-    CustomValidator(validatorFunc, field, data) {
-        let ret = validatorFunc(field, data);
+    static CustomValidator(validatorFunc, field, data) {
+        const ret = validatorFunc(field, data);
         if (typeof ret === 'string') {
```

Comment:
Why we added **static** here. If this is required then do we need to change this code:

```
        case 'custom':
        ret = this.CustomValidator(
            this.entities[i].validators[j].validatorFunc,
            this.entities[i].field,
            data[this.entities[i].field]
        );
        if (ret) {
            return ret;
        }
        break;
    default:
```